package.json

```diff
@@ -1,22 +0,0 @@
-{
```

Comment:
I think this file was accidentally deleted

Reply:
Oh I think this is at the plugin-samples root. Contains the info about the typings dependencies and stuff.

Reply:
Yeah just realized that.

Reply:
I think you might just need to do `git checkout master -- package.json` to get it back